feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-form/checkout-payment-form.component.ts

```diff
@@ -201,22 +233,57 @@ export class CheckoutPaymentFormComponent implements OnInit {
     this.sameAsDeliveryAddress = !this.sameAsDeliveryAddress;
   }
 
-  getAddressCardContent(address: Address): Card {
-    let region = '';
-    if (address.region && address.region.isocode) {
-      region = address.region.isocode + ', ';
+  /**
+   * TODO: (#CXSPA-53) Remove feature check in 6.0
+   */
+  getAddressCardContent(address: Address): Card;
+  getAddressCardContent(address: Address, check: true): Observable<Card>;
+  getAddressCardContent(
+    address: Address,
+    check?: true
```

Comment:
I understand that `check` is a temporary way to switch between returning an observable and object. But this is not descriptive. Please use a more descriptive name and mention the changing of return type in migration docs.

Reply:
change to `returnAsync`